src/constants/enums/facility-type-id.ts

```diff
@@ -0,0 +1,6 @@
+export enum FacilityTypeIdEnum {
+  BSS = '250',
+  EWCS = '260',
+  GEF = '280',
+  // 281 is not used for Facility record, but used for Product id in Loan request.
```

Comment:
Do we need to have 281 in here then? Or no?

Reply:
Maybe we need 2 enums: 
- one for facility records which doesn't have `281`, and
- one for loan requests which does have `281`?

Reply:
This enum doesn't need 281, just wanted to highlight that it is similar to ProductTypeIdEnum.

Improved comment, should be clear now.